### PR TITLE
bots: Use log.html for mvollmer/subscription-manager

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Comment by mvollmer** [comment]
This is a bit of a step in the wrong direction, code wise, I guess, since it puts more config into tests-scan.

File: bots/tests-scan [hunk 1]
```diff
@@ -308,6 +319,9 @@ def prioritize(status, title, labels, priority, context, number):
     state = status.get("state", None)
     update = { "state": "pending" }
 
+    # XXX
+    state = "pending"
+
```
Comment (larskarlitski) [inline bots/tests-scan]: Is this a leftover from debugging?

File: bots/tests-scan [hunk 2]
```diff
@@ -96,6 +96,14 @@ REPO_BRANCH_CONTEXT = {
     }
 }
 
+# Random extra options
+
+extra_options_for_repo = {
+    'mvollmer/subscription-manager': {
+        'tests_invoke_pretty_logs': True
+    }
+}
```
Comment (larskarlitski) [inline bots/tests-scan]: We probably want to name this ALL_CAPS like the other config options above?

Also, could save one jump through a hoop by just giving the additional options for tests-invoke directly?

```python
REPO_EXTRA_OPTIONS = {
    "mvollmer/subscription-manager": [ "--pretty-logs" ]
}
```

File: bots/tests-scan (hunk 1 above)
Comment (mvollmer) [reply bots/tests-scan]: Yep, oops.

File: bots/tests-scan (hunk 2 above)
Comment (mvollmer) [reply bots/tests-scan]: Yes, why not.  Done.

**Comment by mvollmer** [comment]
I have tested this manually like so:
```
$ ./bots/tests-scan -d --repo mvollmer/subscription-manager -p 1
INFO:root:Processing #1 titled 'DBG - expect proper grammar' on revision 1a66c412008a4927079d4fb03332f572856a4d3b
INFO:root:Skipping 'cockpit/rhel-8-0' on #1 because it is pending
PRIORITY=0005 bots/make-checkout --verbose --base=master --repo=mvollmer/subscription-manager pull/1/head 1a66c412008a4927079d4fb03332f572856a4d3b && GITHUB_BASE=mvollmer/subscription-manager TEST_NAME=pull-1-20190531-133131 TEST_REVISION=1a66c412008a4927079d4fb03332f572856a4d3b bots/tests-invoke --pull-number=1  --rebase=master --remote=test --html-logs cockpit/rhel-8-0 pull/1/head
```

And then running this in the repo:
```
$ TEST_PUBLISH=sink-infracloud GITHUB_BASE=mvollmer/subscription-manager TEST_NAME=pull-1-20190531-111212 TEST_REVISION=1a66c412008a4927079d4fb03332f572856a4d3b bots/tests-invoke --pull-number=1 --html-logs cockpit/rhel-8-0 pull/1/head
```

The result can be seen here: https://github.com/mvollmer/subscription-manager/pull/1

**Comment by mvollmer** [comment]
The way this might break is when tests-scan generates a command with `--html-logs` in it and then a old version of tests-invoke is used to execute it.

I _think_ this will not happen: External projects always use bots/ from master, and Cockpit itself will never use that option.

**Pull request merged**